### DISCUSSION TITLE
Thymeleaf Spring support

### DISCUSSION THREAD

**Discussion by mohamed-sarjoon** [thymeleaf/thymeleaf#968] [General]
Hi Team,

We are looking for a replacement of Apache Tiles and came across Thymeleaf. We are trying to integrate thymeleaf with our spring project. 

As per our company's policy, we should only use an active open source software. When checked on thymeleaf-spring project the latest release is on Jan 2022, but we could see the thymeleaf-spring project being moved to thymeleaf project and included in the recent releases.

We just wanted to confirm that the thymeleaf-spring is actively maintained by the community and any security vulnerability on top of it will be addressed by the community. 

Kindly help us by confirming this, so that we will be able to use the thymeleaf.

Regards,
Sarjoon

**Comment by danielfernandez** [comment]
> When checked on thymeleaf-spring project the latest release is on Jan 2022, but we could see the thymeleaf-spring project being moved to thymeleaf project and included in the recent releases.

`thymeleaf-spring` is not a project, but a repository in the Thymeleaf project. As you say, code for the Thymeleaf's Spring integraton modules was moved to this repository.

**Reply by mohamed-sarjoon** [reply]
Thanks @danielfernandez for the confirmation. So it is safe for us to assume that the thymeleaf-spring will be maintained in this repository and  any security vulnerability on top of it will be addressed by the community. Kindly confirm

**Reply by danielfernandez** [reply]
@mohamed-sarjoon I'm confused. Do you want me to sign a support contract or something like that? This is an open source project which, as you can clearly see, is being actively maintained. This project is **not** backed by any software corporation, and anyway even if it were, maintenance responsibilities would be limited to what is established in the Apache License. The Spring integration is a critical part of this project for which, as of the moment, there are no plans to remove. Unfortunately I cannot offer you any other kind of commitment. Regards.

**Reply by mohamed-sarjoon** [reply]
@danielfernandez Thanks for the clarification. Just wanted to know whether the spring integration will be maintained. Got it clarified now
